core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -417,7 +417,6 @@ public Path getWorkingDirectory() {
   public void initialize(URI uri, org.apache.hadoop.conf.Configuration conf) throws IOException {
     super.initialize(uri, conf);
     LOG.debug("initialize({}, {}). Connecting to Alluxio", uri, conf);
-    HadoopUtils.addS3Credentials(conf);
```

Comment:
Is this not needed for s3a?

Reply:
We shouldn't need this even for s3n. This was originally designed for when clients still talked to ufs and you could use the HDFS client to talk to S3.